### DISCUSSION TITLE
Training process

### DISCUSSION THREAD

**Discussion by rubencr14** [RosettaCommons/RFdiffusion#270] [Q&A]
Thank you for this amazing repository and all the effort put into it. I have a question about the training process of the RFdifussion model.

Could you share the code or details used to train the RFdiffusion model? I'm specifically interested in the training loop, loss functions, optimizers, inputs and preprocessing steps.

Thank you!

**Comment by roccomoretti** [comment]
I'm not hooked into the conversation well enough to know when/if the training code will be released, but for details on how things were trained, please see the [original paper](https://www.nature.com/articles/s41586-023-06415-8), particularly the associated Supplementary Information file, which contains many of the details.

Another thing to take a look at is the lectures about RFdiffusion which are available on YouTube. There are several talks - particularly by Joe Watson, David Juergens and Sergey Ovchinnikov - which may provide additional insights on training. (David Baker's talks are usually for more general audiences, and typically do not go into the technical ML details.)
